service-commands/scripts/provision-dev-env.sh

```diff
@@ -113,12 +119,12 @@ function setup_profile() {
     echo 'export AUDIUS_REMOTE_DEV_HOST=$(curl -sfL -H "Metadata-Flavor: Google" http://metadata/computeMetadata/v1/instance/network-interfaces/0/access-configs/0/external-ip)' >> $HOME/.profile
     echo 'export AAO_DIR=$HOME/anti-abuse-oracle' >> $HOME/.profile
     echo 'export TN_DIR=$HOME/trusted-notifier-service' >> $HOME/.profile
-    echo '
```

Comment:
Let's leave this code as is please.

This merely sources files from `~/.aliases/` if `~/.aliases` exists. And it does so at the `~/.profile` level which should be accessible across bash and zsh.